### ISSUE TITLE
box-shadow() problem on subsequent calls

### ISSUE THREAD

**Issue by jhedstrom** [Compass/compass#944]
I'm experiencing an issue where on the first call to `box-shadow()`, the compiled CSS is fine:

```
.foo {
  @include box-shadow(-5px 5px 5px #444);
}
```

results in the expected

```
-webkit-box-shadow: -5px 5px 5px #e1e1df;
-moz-box-shadow: -5px 5px 5px #e1e1df;
box-shadow: -5px 5px 5px #e1e1df;
```

However, the next call to `box-shadow()`

```
.bar {
  @include box-shadow(-5px 5px 5px #444);
}
```

results in

```
.bar {
  -webkit-box-shadow: -5px 5px 5px -5px 5px 5px #e1e1df;
  -moz-box-shadow: -5px 5px 5px -5px 5px 5px #e1e1df;
  box-shadow: -5px 5px 5px -5px 5px 5px #e1e1df;
}
```

Further more, _each_ additional call appends more arguments to the compiled CSS:

```
.baz {
  -webkit-box-shadow: -5px 5px 5px -5px 5px 5px -5px 5px 5px #e1e1df;
  -moz-box-shadow: -5px 5px 5px -5px 5px 5px -5px 5px 5px #e1e1df;
  box-shadow: -5px 5px 5px -5px 5px 5px -5px 5px 5px #e1e1df;
}
```

I'm running compass `0.12.2` and SASS `Sass 3.1.19 (Brainy Betty)` on Ubuntu 12.04.

**Comment by jhedstrom** [comment]
Hmm, apparently the issue was that I had been using a variable named `$shadow` to define the color of the box shadow. Renaming this fixed the issue. However, I am unable to find this on the list of all variables (http://compass-style.org/index/variables/), so perhaps this is a documentation issue?

**Comment by scottdavis** [comment]
you are correct. Its both a documentation error and an issue that the variable name is to generic

https://github.com/chriseppstein/compass/blob/stable/frameworks/compass/stylesheets/compass/css3/_box-shadow.scss#L46

/ping @ericam @imathis

**Comment by jsmestad** [comment]
@scottdavis has this been fixed?

**Comment by scottdavis** [comment]
guess not

**Comment by chriseppstein** [comment]
This got fixed during 1.0
